components/top-area/TopArea.js

```diff
@@ -49,8 +49,7 @@ const TopArea = (props) => {
           <div>
             <div className="grid-flex v-spaced break-xlg">
               <div className="col-4">
-                <Link prefetch href="/tutorials/hello-blockstack">
-                  <a className="g-card code-card dark align-left link">
+                  <a className="g-card code-card dark align-left link" href="https://docs.blockstack.org/browser/hello-blockstack.html">
```

Comment:
```suggestion
                  <a className="g-card code-card dark align-left link" target="_blank" href="https://docs.blockstack.org/browser/hello-blockstack.html">
```
We should have these open in new tabs/windows

Reply:
Sure. I'll add that.